app/views/users/show.html.erb

```diff
@@ -1 +1,13 @@
-<%= "just a placeholder for now" %>
+<h1><%= "Welcome #{current_user.username}!" %></h1>
+<%=  button_to 'Discover Movies' %>
+
+<h3><%= "Friends" %></h3>
+<%=  button_to 'Add Friend' %>
+<% if current_user.followers.count == 0 %>
```

Comment:
Maybe we can make this into a model method 🙂

Reply:
something like current_user.follower_count which will do exactly as you're suggesting to do in line 6